include/chemfiles/files/NcFile.hpp

```diff
@@ -76,6 +76,15 @@ namespace nc {
         void add(count_t start, count_t count, std::vector<float> data);
     };
 
+    class NcDouble final: public NcVariable {
+    public:
+        NcDouble(NcFile& file, netcdf_id_t var) : NcVariable(file, var) {}
+        /// Get `count` values starting at `start` from this variable
+        std::vector<double> get(count_t start, count_t count) const;
+        /// Add `cout` values from `data` starting at `start` in this variable
+        void add(count_t start, count_t count, std::vector<double> data);
```

Comment:
Should this use:
```suggestion
        void add(count_t start, count_t count, const std::vector<double>& data);
```
instead? There is no need to take the vector by value. The same for NcFloat.

Reply:
Makes sense. Done.